tests/UI/pages/FO/hummingbird/myAccount/identity.ts

```diff
@@ -0,0 +1,20 @@
+// Import FO pages
+import {AccountIdentityPage} from '@pages/FO/myAccount/identity';
+
+/**
+ * @class
+ * @extends FOBasePage
+ */
+class AccountIdentity extends AccountIdentityPage {
+  /**
+   * @constructs
+   * Setting up texts and selectors to use
+   */
+  constructor() {
+    super();
+
+    this.theme = 'hummingbird';
```

Comment:
Why not having a this on foBasePage like 
```js
/**** FObasePage.ts ****/
/**
 * FO parent page, contains functions that can be used on all FO page
 * @class
 * @extends CommonPage
 */
export default class FOBasePage extends CommonPage {
constructor(theme:string = 'classic') {
    super();
    this.theme = theme;
...
}


/**** identity.ts ****/
/**
 * @class
 * @extends FOBasePage
 */
class AccountIdentity extends AccountIdentityPage {
  /**
   * @constructs
   * Setting up texts and selectors to use
   */
  constructor() {
    super('hummingbird');
}
```

Reply:
:heavy_check_mark: